src/main/java/org/spongepowered/common/data/util/NbtDataUtil.java

```diff
@@ -41,6 +50,9 @@
     public static final String BLOCK_ENTITY_TAG = "BlockEntityTag";
     public static final String BLOCK_ENTITY_ID = "id";
     public static final String SIGN = "Sign";
+    public static final String DISPLAY = "display";
```

Comment:
Please re-organize these keys since the keys have been organized in [this fashion](https://github.com/SpongePowered/SpongeCommon/blob/master/src/main/java/org/spongepowered/common/data/util/NbtDataUtil.java#L49)